it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
 12557=Errore nell'invio ritardato
 
 ## @name NS_MSG_SAVE_DRAFT_TITLE
-12558=Errore nel salvataggio come bozza
+12558=Errore nel salvare il messaggio come bozza
 
 ## @name NS_MSG_SAVE_TEMPLATE_TITLE
-12559=Errore nel salvataggio come modello
+12559=Errore nel salvare il messaggio come modello
 
 ## @name NS_ERROR_SEND_FAILED_BUT_NNTP_OK
-12560=Il messaggio è stato inviato al gruppo di discussione, ma non all'altro destinatario.
+12560=Il messaggio è stato pubblicato nel newsgroup, ma non è stato inviato ad altri destinatari.
 
 ## @name NS_MSG_ASK_TO_COMEBACK_TO_COMPOSE
-12561=Tornare alla finestra di composizione?
+12561=Vuoi tornare alla finestra di composizione?
 
 ## @name NS_MSG_GENERIC_FAILURE_EXPLANATION
-12562=Verificare che le impostazioni di Posta e gruppi discussione siano corrette e riprovare.
+12562=Verifica che le tue impostazioni account E-mail e Newsgroup siano corrette e riprova.
 
 ## @name NS_MSG_ERROR_READING_FILE
 12563=Errore nella lettura del file.
 
 ## @name NS_FOLLOWUPTO_SENDER
-12564=L'autore di questo messaggio ha richiesto che le risposte siano spedite solamente all'autore. Se si vuole rispondere anche nel gruppo di discussione, aggiungere un nuovo campo indirizzo, scegliere Gruppo discussione tra la lista dei tipi di destinatari, ed inserire il nome del gruppo.
+12564=L'autore di questo messaggio ha richiesto che le risposte siano inviate solamente all'autore. Se anche tu vuoi rispondere nel gruppo di discussione, aggiungi un nuova riga all'area dell'indirizzo, scegli Newsgroup dalla lista dei destinatari ed inserisci il nome del newsgroup.
 
 ## @name NS_MSG_UNDISCLOSED_RECIPIENTS
 ## LOCALIZATION NOTE: this string must be using only US_ASCII characters
-12566=destinatari-ignoti
+12566=destinatari-non-rivelati
 
 ## @name NS_MSG_ERROR_ATTACHING_FILE
-12570=Errore nell'allegare %S. Controllare i permessi di accesso al file.
+12570=Errore nell'allegare %S. Verifica se hai accesso al file.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12570=Errore nell'allegare %S. Verifica di avere accesso al file.